### ISSUE TITLE
[BACK-24] Validate Bot

### ISSUE THREAD

**Issue by sirtobsi** [cmu-dsc/poker-backend-2025#34]
After a bot was uploaded, it needs to be validated by the backend.

AC:

- [] bot validation is triggered after a new bot is uploaded

**Comment by clementou** [comment]
Probably best to run testing suites on the bot in a Lambda function or something.

**Comment by sirtobsi** [comment]
> Probably best to run testing suites on the bot in a Lambda function or something.

yes, sounds good. perhaps we could just save if this check was successful in the BotDao.
